content/blog/2018-12-11-rcites.md

```diff
@@ -0,0 +1,83 @@
+
+To summarize the hackathon: We learn a lot. First, how to create an R package, i.e. creating a structure of files, writing functions, and their documentation, testing functions, checking whether the package is working (it was already a lot at that stage of development). Second, how to make R writing URL requests to query a web API. Third, how to use GitHub for collaborative coding. Of course, we did not become experts on all of these topics, it was just the beginning of an adventure and a lot of work remained, but still … what a day!   
+
+### The long way to perfection   
+
+Although most of the code was written during the conference in Ghent, the package was deeply restructured in the following months. Indeed we realized that a good R package is more than a set of functions and requires a proper organization of the code to avoid redundancy, an adequate unit testing, clear and complete documentation, a cool hexagonal sticker and a lot of thought on how to make the package user-friendly. All this requires time, and we moved slowly while completing the package. After seven months, we were ready to submit to rpenSci and submit a first version (v0.1.0) to CRAN.   
+
+The help of rOpenSci and its onboarding program was pivotal. Two dedicated reviewers, Noam Ross and Margaret Siple, tested the package and suggested enhancements. Their [reviews](https://github.com/ropensci/onboarding/issues/244) were of top-notch quality, they gave extremely valuable advice and were really supportive of our work and the development of the package. After the review, we were very enthusiastic and decided to do our best to address all of their comments. Interestingly enough, during that stage, the size of the code of our key functions shrunk while the number of “private” functions (gathered in zzz.R) increased. We actually added several functionalities while better structuring the code to offer a better user experience (well, we hope so). Now, we can tell that the difference between a working package and a well-crafted package is important. We also understand why rOpenSci suggests doing the CRAN submission after the review: there is no rush for a CRAN release, as the package is already available on GitHub, plus the number of changes after a revision is likely to generate a major release: scripts built based on earlier versions of the package will no longer work with the revised package. This is exactly what happened for us, we released v1.0.0 after the review process, and we’ve learned our lesson.  
+
+### So how does rcites work?   
+
+For a quick illustration, we can use the African bush elephant (Loxodonta africana, hereafter “elephant”). The elephant is a highly endangered species that is not only illegally traded globally but also a flagship species of nature conservation and the logo species of CITES.  
```

Comment:
Important to say what rcites can do, and look back to the intro section where you say what CITES is. These sections should reflect each other to interest the reader. So maybe start by saying here's a question we wanted to answer for this endangered species elephant..